src/pages/studyView/StudyViewPage.tsx

```diff
@@ -362,14 +355,106 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
             )
         } else {
             //TODO: update with loading
-            if(this.store.filteredVirtualStudies.isComplete && !_.isEmpty(this.store.unknownQueriedIds)) {
+            if (this.store.filteredVirtualStudies.isComplete &&
```

Comment:
don't worry about loading behavior here.  i did some work on this in spa branch.

Reply:
should I revert back these changes?

Reply:
no.  i can deal with it on merge